README.md

```diff
@@ -134,7 +134,7 @@ Development
 Ensure you have the latest LTS version of Node.js installed.
 
 Using `yarn` instead of `npm` is recommended. Please see the Yarn [install
-guide](https://yarnpkg.com/docs/install/) if you do not have it already.
+guide](https://yarnpkg.com/getting-started/install) if you do not have it already.
```

Comment:
Hmm, would https://classic.yarnpkg.com/docs/install be simpler to understand perhaps? (We aren't using Yarn 2 yet.)

Reply:
Wasn't aware of that, looks much better! Could you add that you aren't using yarn 2 somewhere as well?

Reply:
We should do that, yes. Perhaps you could do it as part of your change here? Maybe something like edit "Please see the Yarn install guide ..." to "Yarn 1", and then add a new sentence such as "This project has not yet been migrated to Yarn 2, so please ensure `yarn --version` shows a version from the 1.x series."